akka-http-core/src/test/scala/akka/http/impl/model/parser/HttpHeaderSpec.scala

```diff
@@ -228,6 +228,8 @@ class HttpHeaderSpec extends FreeSpec with Matchers {
 
     "Date" in {
       "Date: Wed, 13 Jul 2011 08:12:31 GMT" =!= Date(DateTime(2011, 7, 13, 8, 12, 31))
+// TODO this test fails because we only intend to read this format, not have bidirection generation of it
+//      "Date: Wed, 13-Jul-2011 08:12:31 GMT" =!= Date(DateTime(2011, 7, 13, 8, 12, 31))
```

Comment:
you can simply add a `.renderedTo(...)` clause at the very end to get around the problem you describe in the `TODO`.
The subsequent example shows how.